### ISSUE TITLE
AVT 3 - Carbon X Slider does not announce status correct using a screen reader

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1829]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)
<and>
Windows 7 
Chrome version 72.0.3626.96 (Official Build) (64-bit)
JAWS

## Description
AVT 3 - Slider status numbers are announced incorrectly with VoiceOver and not announced at all with JAWS. The number input field is announced as a stepper which makes it difficult to understand the field that is focused. 

The example should follow the keyboard interaction and ARIA attributes outlined in the ARIA Authoring Practices Horizontal Slider Design Pattern.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.

**Comment by snidersd** [comment]
@dakahn Retested with VoiceOver and this is still an issue for the Vanilla component.

**Comment by dakahn** [comment]
@snidersd tested in the VoiceOver in Safari and it seems like VO is maybe bugged?
https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/slider/slider-1.html
Testing the WAI-ARIA example and the controls for moving the slider don't work and the slider itself reads poorly. Tested with JAWS and seems pretty great? Can you confirm?

**Comment by snidersd** [comment]
@dakahn  I tested VO on macOS with Safari and Chrome and get the same results. JAWS works for both the Carbon React Component. I also tested the ARIA slider example with VO and Safari on iOS and it works, but it does not work with the React Carbon slider, so I can't confirm that its VO or an issue related to the macOS.

**Comment by joshblack** [comment]
Marking as blocked until we can confirm where the underlying issue resides. Also ran into what @dakahn mentioned in VO on Safari:

 
![demo](https://user-images.githubusercontent.com/3901764/69451987-4b367580-0d26-11ea-8215-3d1118d4409e.gif)

Procedure

1. Navigate to Slider
2. Press Control+Option+Shift+Down Arrow
3. Press Control+Option+Right Arrow

Expected: Slider to increment value
Actual: Slider value remains the same

**Comment by dakahn** [comment]
Since this is WCAG AAA requirements and we shoot for AA i'm going to turn this into an open proposal/enhancement.

**Comment by dakahn** [comment]
It seems like this might be a bug on Safari's end. Using the [WAI-ARIA slider](https://w3c.github.io/aria-practices/examples/slider/slider-1.html) example as a neutral test case we see the same problem (unable to move slider and have number read reliably). There's a bug filed in the Webkit bug tracker we'll watch for updates.

**Comment by tw15egan** [comment]
@dakahn have there been any updates on this?

**Comment by dakahn** [comment]
Tested and works as intended on JAWS on Firefox latest and VoiceOver on Safari. Looks like this one got patched up without us. Closing, but can reopen if I missed something. 👍🏽 🏄🏽
